cumulus_etl/etl/studies/covid_symptom/covid_ctakes.py

```diff
@@ -86,20 +86,28 @@ def is_covid_match(m: ctakesclient.typesystem.MatchText):
         logging.warning("Could not check polarity for docref %s (%s): %s", docref_id, type(exc).__name__, exc)
         return None
 
+    # Helper to make a single row (match_value is None if there were no found symptoms at all)
+    def _make_covid_symptom_row(row_id: str, match: dict | None) -> dict:
+        return {
+            "id": row_id,
+            "docref_id": docref_id,
+            "encounter_id": encounter_id,
+            "subject_id": subject_id,
+            "generated_on": timestamp,
+            "task_version": task_version,
+            "match": match,
+        }
+
     # Now filter out any non-positive matches
     positive_matches = []
     for i, match in enumerate(matches):
         if polarities_cnlp[i] == ctakesclient.typesystem.Polarity.pos:
-            positive_matches.append(
-                {
-                    "id": f"{docref_id}.{i}",
-                    "docref_id": docref_id,
-                    "encounter_id": encounter_id,
-                    "subject_id": subject_id,
-                    "generated_on": timestamp,
-                    "task_version": task_version,
-                    "match": match.as_json(),
-                }
-            )
+            positive_matches.append(_make_covid_symptom_row(f"{docref_id}.{i}", match.as_json()))
+
+    if not positive_matches:
+        # In this case, we write out a single row with a null match,
+        # to flag to the downstream SQL that this DocRef was processed,
+        # it just had no symptoms (vs a DocRef that we never handled).
+        positive_matches = [_make_covid_symptom_row(f"{docref_id}.0", None)]
```

Comment:
1. I tested with a Synthea dataset that rows with null matches don't affect our current covid_symptom SQL code at all (they are ignored). So this is harmless and will let us write SQL in future that does care about these docs.
2. The SQL schema for this table is provided manually by the ETL (not inferred), so the SQL will always know the schema, even if for some weird reason all the `match` columns are null.